src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -0,0 +1,251 @@
+        type: identity.type,
+        groups: identity.groups?.length || 0,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  const selectedIdentities = identities.filter((identity) =>
+    selectedIdentityNames.includes(identity.name),
+  );
+
+  // NOTE: tls user group membership cannot be modified, this will be supported in the future
+  const nonTlsUsers = identities.filter((identity) => {
+    const isTlsIdentity = identity.authentication_method === "tls";
+    return !isTlsIdentity;
+  });
+
+  if (isLoading) {
+    return <Loader text="Loading identities" />;
+  }
+
+  return (
+    <>
+      <CustomLayout
+        contentClassName="u-no-padding--bottom"
+        header={
+          <PageHeader>
+            <PageHeader.Left>
+              <PageHeader.Title>
+                <HelpLink
+                  href={`${docBaseLink}/explanation/authorization`}
+                  title="Learn more about permissions"
+                >
+                  Identities
+                </HelpLink>
+              </PageHeader.Title>
+              {!selectedIdentityNames.length && !panelParams.panel && (
+                <PageHeader.Search>
+                  <PermissionIdentitiesFilter />
+                </PageHeader.Search>
+              )}
+              {!!selectedIdentityNames.length && (
+                <PermissionIdentityEditGroupsBtn
+                  identities={selectedIdentities}
+                  className="u-no-margin--bottom"
+                />
+              )}
+            </PageHeader.Left>
+          </PageHeader>
+        }
+      >
+        <NotificationRow />
+        <Row className="permission-identities">
+          <ScrollableTable
+            dependencies={[identities]}
+            tableId="identities-table"
+            belowIds={["status-bar"]}
+          >
+            <TablePagination
+              data={sortedRows}
+              id="pagination"
+              itemName="user"
+              className="u-no-margin--top"
+              aria-label="Table pagination control"
+              description={
+                selectedIdentityNames.length > 0 && (
+                  <SelectedTableNotification
+                    totalCount={nonTlsUsers.length ?? 0}
+                    itemName="user"
+                    parentName="server"
+                    selectedNames={selectedIdentityNames}
+                    setSelectedNames={setSelectedIdentityNames}
+                    filteredNames={nonTlsUsers.map((item) => item.name)}
+                  />
+                )
+              }
+            >
+              <SelectableMainTable
+                id="identities-table"
+                headers={headers}
+                rows={sortedRows}
+                sortable
+                emptyStateMsg="No identities found matching this search"
+                onUpdateSort={updateSort}
+                itemName="user"
```

Comment:
```suggestion
                itemName="identities"
```
Staying consistent?

Reply:
this might require some upstreaming for the `TablePagination` component since plural for identity isn't a catered for scenario :sweat_smile:

Reply:
strange, I remember changing this, it's definitely fixed now